src/server/on-off-farm/controller.test.js

```diff
@@ -0,0 +1,64 @@
+import { createServer } from '~/src/server/index.js'
+import { statusCodes } from '~/src/server/common/constants/status-codes.js'
+
+describe('#onOffFarmController', () => {
+  /** @type {Server} */
+  let server
+
+  beforeAll(async () => {
+    server = await createServer()
```

Comment:
During test, are we using persistent state (e.g. redis running on docker)?  As the service expands, we may want to start mocking out parts of this, but I'm definitely okay for the moment going with that.

Reply:
Catch up with you later on how the cache works - it will be using an in memory cache here i suspect